src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js

```diff
@@ -417,9 +417,11 @@ function ImpBookTab({
                 </Grid>
 
                 <Grid item xs={6}>
-                    {/* todo implement delivery term code stuff  */}
                     <Dropdown
-                        items={[]}
+                        items={deliveryTerms.map(e => ({
```

Comment:
might be nice to put these items in a variable above

Reply:
Do you mean map them somewhere else, like in the container when they're selected? Or to store them in something like a localDeliveryTerms variable? I'd prefer not to store them locally as I've got a few like this and adding consts & useEffects feels a bit cluttered when it's not adding anything

Reply:
i never thought of doing it in the container. That actually might be quite nice if the items aren't used for anything else. I just meant in a const as i think it makes the dropdown declaration a bit cleaner.

Reply:
+1 for mapping arrays passed as props into the form actually used by the component in the container

Reply:
Yeah I've got a few where the items aren't used for anything except dropdowns, I'll move the mappings to the container and then can just have nice "items={deliveryTerms}" in the components